package.json

```diff
@@ -25,10 +25,9 @@
 		"eslint-plugin-vitest-globals": "^1.5.0",
 		"husky": "^9.0.11",
 		"jsdom": "^24.0.0",
-		"lint-staged": "^15.2.2",
```

Comment:
`lint-staged` is not needed since it's being called with `npx` in the precommit hook - https://github.com/luisaugusto/smart-shopping-list/blob/update-packages-05-22/.husky/pre-commit#L1